### DISCUSSION TITLE
Basic functionality and parsing

### DISCUSSION THREAD

**Discussion by schaffung** [srijan-sivakumar/odinControl#21] [General]
@aujjwal-redhat  please check out the updated functionalities. Do go through the readme and let me know if more clarification is required.

For now, I'd say do look over couple of things while using this module
1. Let's not use at as a separate package, so directly clone and copy the rexe.py
2. Validation for connection failures is not working as intended. I'd say try out with servers which are up and running.

**Comment by aujjwal-redhat** [comment]
Readme looks good. Quite informative explanation.

Parsing: #22 faced the issue.
I Will see if it needs any modification and submit a patch.
